ethcore/src/verification/queue/kind.rs

```diff
@@ -119,14 +119,13 @@ pub mod blocks {
 
 	impl Unverified {
 		/// Create an `Unverified` from raw bytes.
-		pub fn new(bytes: Bytes) -> Self {
-			use views::BlockView;
+		pub fn from_rlp(bytes: Bytes) -> Result<Self, ::rlp::DecoderError> {
 
-			let header = view!(BlockView, &bytes).header();
-			Unverified {
+			let header : Header = ::rlp::Rlp::new(&bytes).val_at(0)?; 
```

Comment:
Drop the space before the colon `let header: Header`.

Reply:
I've removed the type annotation since it was redundant anyway.